scripts/lmdk/tools/header_pack.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+import json
+import pathlib
+import shutil
+
+
+# Headers for needs of lmdk are defined in
+#            lmdk/include/headers_list.json
+
+
+SOF_TOP = pathlib.Path(__file__).parents[3].resolve()
+LMDK_HEADERS = SOF_TOP / "lmdk" / "include" / "headers_manifest.json"
+
+
+def create_separate_headers():
+    f = open(LMDK_HEADERS)
+    data = json.load(f)
+    src = ''
+
+    for i in data:
+        src = ''
+        for p in i[:-1]:
+            src += p
+            src += "/"
+        pathlib.Path(SOF_TOP / "lmdk" / "include" / "sof" / src).mkdir(parents=True, exist_ok=True)
```

Comment:
SOF_TOP is already a Path
```suggestion
        (SOF_TOP / "lmdk" / "include" / "sof" / src).mkdir(parents=True, exist_ok=True)
```